src/ui/common/src/components/integrations/dialogs/dialog.tsx

```diff
@@ -330,5 +277,10 @@ export function isConfigComplete(
     return true;
   }
 
-  return Object.values(config).every((x) => x === undefined || (x && x !== ''));
+  return (
```

Comment:
This code is fun :) From looking at it, I think that it is checking to see if each of the config objects have some key set?

Perhaps we should add a small comment here for future people who come across this code about it's intended behavior.

Reply:
exactly..but yes I agree we should have some comments here

Reply:
actually should be sufficient to just do `.every(x => !!x)`